apps/spruce/src/components/Header/AuxiliaryDropdown.tsx

```diff
@@ -19,6 +22,23 @@ export const AuxiliaryDropdown: React.FC<AuxiliaryDropdownProps> = ({
   const { sendEvent } = useNavbarAnalytics();
   const { distro } = useFirstDistro();
 
+  const { betaFeatures } = useMergedBetaFeatures();
+  const { spruceWaterfallEnabled } = betaFeatures ?? {};
+
+  const inverseLink = spruceWaterfallEnabled
+    ? {
+        "data-cy": "auxiliary-dropdown-project-health",
+        text: "Project Health",
+        to: getCommitsRoute(projectIdentifier),
+        onClick: () => sendEvent({ name: "Clicked project health link" }),
+      }
+    : {
+        "data-cy": "auxiliary-dropdown-waterfall",
+        text: "Waterfall",
```

Comment:
What do you think of something a bit more eye catching? Would just need to update the `text` prop to be React.ReactNode
<img width="240" alt="image" src="https://github.com/user-attachments/assets/6a7102cc-baf3-4ee0-a5e2-710a3fd7881f" />

```suggestion
        text: (
          <span>
            Waterfall{" "}
            <Badge darkMode variant={Variant.Blue}>
              Beta
            </Badge>
          </span>
        ),
```

Reply:
did you want the beta badge on the primary link as well? i wasn't sure :O i can just remove if not!

Reply:
Sorry, not to be too nitpicky but imo no. I feel like the one in the dropdown menu might function as a bit of a call to action/intrigue for users who haven't opted in yet.